on_device_training/mobilevit/mobilevit_console/mobilevit_console/Program.cs

```diff
@@ -0,0 +1,168 @@
+
+        static string CHECKPOINTPATH = Path.Combine(PARENTDIR, "checkpoint");
+        static string TRAININGMODELPATH = Path.Combine(PARENTDIR, "training_model.onnx");
+        static string EVALMODELPATH = Path.Combine(PARENTDIR, "eval_model.onnx");
+        static string OPTIMIZERMODELPATH = Path.Combine(PARENTDIR, "optimizer_model.onnx");
+
+        static string TRAINEDMODELFILE = "trained_mobilevit.onnx";
+
+        static string DEFAULTINFERENCE1 = Path.Combine(PARENTDIR, "test1.png");
+        static string DEFAULTINFERENCE2 = Path.Combine(PARENTDIR, "test3.png");
+        
+        static void Main(string[] args)
+        {
+            if (args.Length < 1)
+            {
+                Console.Write("Please supply the file path to the FER dataset as the first command line argument.");
+                System.Environment.Exit(1);
+            }
+
+            string dataPath = args[0];
+            DataLoader dl = loadDataset(dataPath);
+            CreateAndRunTrainingSession(8, 8, dl);
+            Console.WriteLine("Finished training and exporting");
+
+            string inferencePath = Path.Combine(Directory.GetCurrentDirectory(), TRAINEDMODELFILE);
+            var inferenceSession = new InferenceSession(inferencePath);
+
+            if (args.Length > 1)
+            {
+                // assume additional arguments are paths to inference images
+                for (int i = 1; i < args.Length; i++)
+                {
+                    runInferenceSession(inferenceSession, dl, args[i]);
+                }
+            }
+            else
+            {
+                runInferenceSession(inferenceSession, dl, DEFAULTINFERENCE1);
+                runInferenceSession(inferenceSession, dl, DEFAULTINFERENCE2);
+            }
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="pathToFER">Path to the FER directory containing 7 unzipped folders labelled with 
+        /// their corresponding emotion</param>
+        /// <param name="trimDatasetTo">Optional parameter to trim the size of the dataset. If left as -1, 
+        /// no trimming will happen and the full dataset will be used for training.</param>
+        /// <returns></returns>
+        public static DataLoader loadDataset(string pathToFER, int trimDatasetTo = -1)
+        {
+            var watch = System.Diagnostics.Stopwatch.StartNew();
+            DataLoader dl = new DataLoader();
+            dl.loadFER(pathToFER);
+            if (trimDatasetTo > 0)
+            {
+                dl.trimFER(trimDatasetTo);
+            }
+
+            watch.Stop();
+            Console.WriteLine($"Execution Time for loading images and shuffling: {watch.ElapsedMilliseconds} ms");
+
+            return dl;
+        }
+
+        public static void CreateAndRunTrainingSession(int numEpochs, int batchSize, DataLoader dl)
```

Comment:
Shall we keep the naming consistent? `createAndRunTrainingSession` vs. `CreateAndRunTrainingSession`?